apps/scan/backend/src/electrical_testing/app.ts

```diff
@@ -108,10 +109,18 @@ function buildApi({
     },
 
     async getLatestScannedSheet(): Promise<SheetOf<string> | null> {
-      const allScannedImageNames = await readdir(workspace.ballotImagesPath);
-      const latestElectricalTestingImageName = iter(allScannedImageNames)
-        .filter((name) => name.startsWith('electrical-testing-'))
-        .max((a, b) => a.localeCompare(b));
+      const usbDriveStatus = await usbDrive.status();
+      const basedir =
+        usbDriveStatus.status === 'mounted'
+          ? join(usbDriveStatus.mountPoint, 'ballot-images')
```

Comment:
Curious - what does this addition give? Wondering if it's adding production machine support, and previously it was set up for dev workspace

Reply:
Another change in #6380 was to write the images to the USB drive instead of the dev workspace if the USB drive is mounted. This updates the code that looks for the images to reflect that.